client/styles/components/_editor.scss

```diff
@@ -85,9 +85,9 @@
 */
 
 .CodeMirror-dialog {
-  position: absolute;
+  position: fixed;
```

Comment:
This styling isn't working for me. Given that screen widths are different on different devices, I would use a percent value rather than a fixed pixel value. Locally, I've changed this to
```scss
.CodeMirror-dialog {
  position: fixed;
  top: 0;
  left: 50%;
  margin-left: - #{365/2/$base-font-size}rem;

  z-index: 10;

  width: 100%;
  max-width: #{365 / $base-font-size}rem;
 // ... the rest of the styles
}
```